### PR TITLE
test_ssa_templates: skip 5.4 due to bz 1230248

### PR DESCRIPTION
**Pull request by vrutkovs**
This also removes settings creds for host/datastore for RHOS

{{pytest: cfme/tests/infrastructure/test_instance_analysis.py -k 'test_ssa_template' --use-provider='rhos7-ga' --long-running -v}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 7a71ad0d9a7024ff8cbd16bd8780f868dbb0407f:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d6ec4c5b45845ec5961f8f78e9dd5c4d59e27663:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1592067a3ca258143cd2dc6583c28d1139e014fe:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 42f4d5fe030e616155e3d9b0b0adbadcdc866680:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 28acbbb212c40bab791cc0406ccc82f2ea667f52:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/common/vm.py [hunk 1]
```diff
@@ -285,7 +285,7 @@ def find_quadicon(
             elif self.is_vm:
                 self.provider.load_all_provider_vms()
             else:
-                self.provider.load_all_provider_templates()
+                self.provider.load_all_provider_images()
```
Comment (mfalesni) [inline cfme/common/vm.py]: Why changing this? On line 521 in `cfme/common/providers.py`, it is just an alias for `load_all_provider_templates`.

Comment (vrutkovs) [inline cfme/common/vm.py]: Hmm, weird, I though I reverted this :) Will kill it in next commit

**Comment by dajoRH** [comment]
Lint report for commit 4d339d6af7bae0b259b1ed4ce5ba9afe9a543264:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/configure/configuration/__init__.py [hunk 2]
```diff
@@ -1842,7 +1842,7 @@ def get_server_roles(navigate=True, db=True):
         role_list = {}
         for (name, locator) in server_roles.fields:
             try:
-                role_list[name] = sel.element(locator).is_selected()
+                role_list[name] = sel.element(locator.locate()).is_selected()
```
Comment (psav) [inline cfme/configure/configuration/__init__.py]: Why the ```locate()``` call here? I thought ```sel.element``` called locate as a first effort anyway?

**Comment by dajoRH** [comment]
Lint report for commit d89b6ebcd7bd0d49c857f5791f9ba2340f543a74:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0faad5eb1ad7e2f3c72907ed01d3a2d73c082d0e:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by vrutkovs** [comment]
@psav @mfalesni plz review.

Nightly and Upstream are affected by a weird issue - roles cannot be set correctly, not sure if its caused by internal code changes or app code

**Pull request merged**